apidocs/overview.md

```diff
@@ -0,0 +1,438 @@
+        {
+            "kind": "removeUserTargets",
+            "values": ["438580d8-02ee-418d-9eec-0085cab2bdf0"],
+            "variationId": "852cb784-54ff-46b9-8c35-5498d2e4f270"
+        },
+        {
+            "kind": "addUserTargets",
+            "values": ["438580d8-02ee-418d-9eec-0085cab2bdf0"],
+            "variationId": "1bb18465-33b6-49aa-a3bd-eeb6650b33ad"
+        }
+    ]
+}
+```
+
+<blockquote>
+    <h3><span>📘</span>Supported semantic patch API endpoints</h3>
+    <p>TODO: update these links</p>
+    <p><a href="#operation/patchFeatureFlag">Update feature flag</a></p>
+    <p><a href="">Update expiring user targets on feature flag</a></p>
+    <p><a href="">Update expiring user target for flags</a></p>
+</blockquote>
+
+# Errors
+
+The API always returns errors in a common format. Here's an example:
+
+```json
+{
+    "code": "invalid_request",
+    "message": "A feature with that key already exists",
+    "id": "30ce6058-87da-11e4-b116-123b93f75cba"
+}
+```
+
+The general class of error is indicated by the `code`. The `message` is a human-readable explanation of what went wrong. The `id` is a unique identifier-- use it when you're working with LaunchDarkly support to debug a problem with a specific API call.
+
+## HTTP Status - Error Response Codes
+
+| Code | Definition | Desc. | Possible Solution |
+| ---- | ---------- | ----- | ----------------- |
+| 400  | Bad Request | A request that fails may return this HTTP response code | Ensure JSON syntax in request body is correct. |
+| 401  | Unauthorized | User doesn't have permission to an API call | Ensure your SDK key is good. |
+| 403  | Forbidden | User does not have permission for operation. | Ensure that the user or access token has proper permissions set. |
+| 409  | Conflict | The API request could not be completed because it conflicted with a concurrent API request. | Retry your request. |
+| 429  | Too many requests | See [Rate limiting](ref:rate-limiting) | Wait and try again later. |
+
+# CORS
+
+The LaunchDarkly API supports Cross Origin Resource Sharing (CORS) for AJAX requests from any origin.  If an `Origin` header is given in a request, it will be echoed as an explicitly allowed origin. Otherwise, a wildcard will be returned: `Access-Control-Allow-Origin: *`.  For more information on CORS, see the [CORS W3C Recommendation](http://www.w3.org/TR/cors).  Example CORS headers might look like:
+
+```http
+Access-Control-Allow-Headers: Accept, Content-Type, Content-Length, Accept-Encoding, Authorization
+Access-Control-Allow-Methods: OPTIONS, GET, DELETE, PATCH
+Access-Control-Allow-Origin: *
+Access-Control-Max-Age: 300
+```
+
+You can make authenticated CORS calls just as you would make same-origin calls, using either [token or session-based authentication](doc:authentication). If you’re using session auth, you should set the `withCredentials` property for your `xhr` request to `true`. Remember-- you should never expose your access tokens to untrusted users.
```

Comment:
```suggestion
You can make authenticated CORS calls just as you would make same-origin calls, using either [token or session-based authentication](doc:authentication). If you’re using session auth, you should set the `withCredentials` property for your `xhr` request to `true`. You should never expose your access tokens to untrusted users.
```
As a style choice we avoid using "please note," "remember," "don't forget," etc. as they are extraneous.